index.src.html

```diff
@@ -783,11 +797,10 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
 callback ReportingObserverCallback = void (sequence&lt;Report> reports, ReportingObserver observer);
 
 dictionary ReportingObserverOptions {
-  sequence&lt;DOMString> types;
+    sequence&lt;DOMString> types;
+    boolean futureOnly;
```

Comment:
`futureOnly` being not present and being specified as false should behave identically, right?  You can make this clear and simplify some spec text by specifying a default value (`= false`).  See [here](https://heycam.github.io/webidl/#dfn-dictionary).

Reply:
Okay, done.